src/libraries/Microsoft.PowerFx.Connectors/Internal/ConnectorExtensions.cs

```diff
@@ -10,7 +10,7 @@ internal class ConnectorExtensions
         internal string Summary;
         internal bool ExplicitInput;
 
-        internal ConnectorExtensions(IOpenApiExtensible extension, IOpenApiExtensible body, bool numberIsFloat)
```

Comment:
>numberIsFloat [](http://example.com/codeflow?start=97&length=13)

this would be a break if we had new Cx with old Fx.

But I think we passed that window and no longer think we need to maintain that.